[2024-11-18T19:42:24.846+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spotify_recommender_dag_v3.preprocess_and_split manual__2024-11-18T19:37:12.494221+00:00 [queued]>
[2024-11-18T19:42:24.851+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spotify_recommender_dag_v3.preprocess_and_split manual__2024-11-18T19:37:12.494221+00:00 [queued]>
[2024-11-18T19:42:24.851+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-11-18T19:42:24.860+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): preprocess_and_split> on 2024-11-18 19:37:12.494221+00:00
[2024-11-18T19:42:24.863+0000] {standard_task_runner.py:60} INFO - Started process 1333 to run task
[2024-11-18T19:42:24.864+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'spotify_recommender_dag_v3', 'preprocess_and_split', 'manual__2024-11-18T19:37:12.494221+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/music_process.py', '--cfg-path', '/tmp/tmplzy046oh']
[2024-11-18T19:42:24.865+0000] {standard_task_runner.py:88} INFO - Job 114: Subtask preprocess_and_split
[2024-11-18T19:42:24.892+0000] {task_command.py:423} INFO - Running <TaskInstance: spotify_recommender_dag_v3.preprocess_and_split manual__2024-11-18T19:37:12.494221+00:00 [running]> on host 98017b53a184
[2024-11-18T19:42:25.678+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mart√≠n Horn' AIRFLOW_CTX_DAG_ID='spotify_recommender_dag_v3' AIRFLOW_CTX_TASK_ID='preprocess_and_split' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T19:37:12.494221+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-18T19:37:12.494221+00:00'
[2024-11-18T19:42:25.680+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvys0pjqq0 --system-site-packages --python=python
[2024-11-18T19:42:25.684+0000] {process_utils.py:186} INFO - Output:
[2024-11-18T19:42:26.123+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 207ms
[2024-11-18T19:42:26.124+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venvys0pjqq0, clear=False, no_vcs_ignore=False, global=True)
[2024-11-18T19:42:26.124+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-11-18T19:42:26.125+0000] {process_utils.py:190} INFO -     added seed packages: pip==24.2, setuptools==75.2.0, wheel==0.44.0
[2024-11-18T19:42:26.125+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-11-18T19:42:26.141+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvys0pjqq0/bin/pip install -r /tmp/venvys0pjqq0/requirements.txt
[2024-11-18T19:42:26.145+0000] {process_utils.py:186} INFO - Output:
[2024-11-18T19:42:27.002+0000] {process_utils.py:190} INFO - Collecting boto3==1.28.0 (from -r /tmp/venvys0pjqq0/requirements.txt (line 1))
[2024-11-18T19:42:27.005+0000] {process_utils.py:190} INFO -   Using cached boto3-1.28.0-py3-none-any.whl.metadata (6.6 kB)
[2024-11-18T19:42:27.006+0000] {process_utils.py:190} INFO - Requirement already satisfied: pandas==2.0.3 in /home/***/.local/lib/python3.8/site-packages (from -r /tmp/venvys0pjqq0/requirements.txt (line 2)) (2.0.3)
[2024-11-18T19:42:27.007+0000] {process_utils.py:190} INFO - Requirement already satisfied: scikit-learn==1.3.2 in /home/***/.local/lib/python3.8/site-packages (from -r /tmp/venvys0pjqq0/requirements.txt (line 3)) (1.3.2)
[2024-11-18T19:42:27.397+0000] {process_utils.py:190} INFO - Collecting botocore<1.32.0,>=1.31.0 (from boto3==1.28.0->-r /tmp/venvys0pjqq0/requirements.txt (line 1))
[2024-11-18T19:42:27.399+0000] {process_utils.py:190} INFO -   Using cached botocore-1.31.85-py3-none-any.whl.metadata (6.1 kB)
[2024-11-18T19:42:27.401+0000] {process_utils.py:190} INFO - Requirement already satisfied: jmespath<2.0.0,>=0.7.1 in /home/***/.local/lib/python3.8/site-packages (from boto3==1.28.0->-r /tmp/venvys0pjqq0/requirements.txt (line 1)) (0.10.0)
[2024-11-18T19:42:27.424+0000] {process_utils.py:190} INFO - Collecting s3transfer<0.7.0,>=0.6.0 (from boto3==1.28.0->-r /tmp/venvys0pjqq0/requirements.txt (line 1))
[2024-11-18T19:42:27.426+0000] {process_utils.py:190} INFO -   Using cached s3transfer-0.6.2-py3-none-any.whl.metadata (1.8 kB)
[2024-11-18T19:42:27.448+0000] {process_utils.py:190} INFO - Requirement already satisfied: python-dateutil>=2.8.2 in /home/***/.local/lib/python3.8/site-packages (from pandas==2.0.3->-r /tmp/venvys0pjqq0/requirements.txt (line 2)) (2.8.2)
[2024-11-18T19:42:27.448+0000] {process_utils.py:190} INFO - Requirement already satisfied: pytz>=2020.1 in /home/***/.local/lib/python3.8/site-packages (from pandas==2.0.3->-r /tmp/venvys0pjqq0/requirements.txt (line 2)) (2023.3.post1)
[2024-11-18T19:42:27.449+0000] {process_utils.py:190} INFO - Requirement already satisfied: tzdata>=2022.1 in /home/***/.local/lib/python3.8/site-packages (from pandas==2.0.3->-r /tmp/venvys0pjqq0/requirements.txt (line 2)) (2023.4)
[2024-11-18T19:42:27.449+0000] {process_utils.py:190} INFO - Requirement already satisfied: numpy>=1.20.3 in /home/***/.local/lib/python3.8/site-packages (from pandas==2.0.3->-r /tmp/venvys0pjqq0/requirements.txt (line 2)) (1.24.4)
[2024-11-18T19:42:27.454+0000] {process_utils.py:190} INFO - Requirement already satisfied: scipy>=1.5.0 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venvys0pjqq0/requirements.txt (line 3)) (1.10.1)
[2024-11-18T19:42:27.455+0000] {process_utils.py:190} INFO - Requirement already satisfied: joblib>=1.1.1 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venvys0pjqq0/requirements.txt (line 3)) (1.4.2)
[2024-11-18T19:42:27.455+0000] {process_utils.py:190} INFO - Requirement already satisfied: threadpoolctl>=2.0.0 in /home/***/.local/lib/python3.8/site-packages (from scikit-learn==1.3.2->-r /tmp/venvys0pjqq0/requirements.txt (line 3)) (3.5.0)
[2024-11-18T19:42:27.456+0000] {process_utils.py:190} INFO - Requirement already satisfied: urllib3<1.27,>=1.25.4 in /home/***/.local/lib/python3.8/site-packages (from botocore<1.32.0,>=1.31.0->boto3==1.28.0->-r /tmp/venvys0pjqq0/requirements.txt (line 1)) (1.26.18)
[2024-11-18T19:42:27.459+0000] {process_utils.py:190} INFO - Requirement already satisfied: six>=1.5 in /home/***/.local/lib/python3.8/site-packages (from python-dateutil>=2.8.2->pandas==2.0.3->-r /tmp/venvys0pjqq0/requirements.txt (line 2)) (1.16.0)
[2024-11-18T19:42:27.564+0000] {process_utils.py:190} INFO - Using cached boto3-1.28.0-py3-none-any.whl (135 kB)
[2024-11-18T19:42:27.566+0000] {process_utils.py:190} INFO - Using cached botocore-1.31.85-py3-none-any.whl (11.3 MB)
[2024-11-18T19:42:27.579+0000] {process_utils.py:190} INFO - Using cached s3transfer-0.6.2-py3-none-any.whl (79 kB)
[2024-11-18T19:42:28.381+0000] {process_utils.py:190} INFO - Installing collected packages: botocore, s3transfer, boto3
[2024-11-18T19:42:28.381+0000] {process_utils.py:190} INFO -   Attempting uninstall: botocore
[2024-11-18T19:42:28.382+0000] {process_utils.py:190} INFO -     Found existing installation: botocore 1.35.44
[2024-11-18T19:42:28.382+0000] {process_utils.py:190} INFO -     Not uninstalling botocore at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venvys0pjqq0
[2024-11-18T19:42:28.382+0000] {process_utils.py:190} INFO -     Can't uninstall 'botocore'. No files were found to uninstall.
[2024-11-18T19:42:28.864+0000] {process_utils.py:190} INFO -   Attempting uninstall: s3transfer
[2024-11-18T19:42:28.865+0000] {process_utils.py:190} INFO -     Found existing installation: s3transfer 0.10.3
[2024-11-18T19:42:28.866+0000] {process_utils.py:190} INFO -     Not uninstalling s3transfer at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venvys0pjqq0
[2024-11-18T19:42:28.866+0000] {process_utils.py:190} INFO -     Can't uninstall 's3transfer'. No files were found to uninstall.
[2024-11-18T19:42:28.888+0000] {process_utils.py:190} INFO -   Attempting uninstall: boto3
[2024-11-18T19:42:28.889+0000] {process_utils.py:190} INFO -     Found existing installation: boto3 1.35.44
[2024-11-18T19:42:28.889+0000] {process_utils.py:190} INFO -     Not uninstalling boto3 at /home/***/.local/lib/python3.8/site-packages, outside environment /tmp/venvys0pjqq0
[2024-11-18T19:42:28.889+0000] {process_utils.py:190} INFO -     Can't uninstall 'boto3'. No files were found to uninstall.
[2024-11-18T19:42:28.925+0000] {process_utils.py:190} INFO - ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
[2024-11-18T19:42:28.926+0000] {process_utils.py:190} INFO - aiobotocore 2.9.0 requires botocore<1.33.14,>=1.33.2, but you have botocore 1.31.85 which is incompatible.
[2024-11-18T19:42:28.927+0000] {process_utils.py:190} INFO - Successfully installed boto3-1.28.0 botocore-1.31.85 s3transfer-0.6.2
[2024-11-18T19:42:28.991+0000] {process_utils.py:190} INFO - 
[2024-11-18T19:42:28.992+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 24.2 -> 24.3.1
[2024-11-18T19:42:28.993+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2024-11-18T19:42:29.131+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvys0pjqq0/bin/python /tmp/venv-callk_pz738u/script.py /tmp/venv-callk_pz738u/script.in /tmp/venv-callk_pz738u/script.out /tmp/venv-callk_pz738u/string_args.txt /tmp/venv-callk_pz738u/termination.log
[2024-11-18T19:42:29.135+0000] {process_utils.py:186} INFO - Output:
[2024-11-18T19:42:31.192+0000] {process_utils.py:190} INFO - Scaler saved to MinIO.
[2024-11-18T19:42:31.419+0000] {python.py:201} INFO - Done. Returned value was: Data preprocessed and saved.
[2024-11-18T19:42:31.433+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=spotify_recommender_dag_v3, task_id=preprocess_and_split, execution_date=20241118T193712, start_date=20241118T194224, end_date=20241118T194231
[2024-11-18T19:42:31.445+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-18T19:42:31.454+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
